mailpoet/lib/EmailEditor/Engine/Renderer/BlockRenderer.php

```diff
@@ -2,6 +2,28 @@
 
 namespace MailPoet\EmailEditor\Engine\Renderer;
 
-interface BlockRenderer {
-  public function render(string $blockContent, array $parsedBlock): string;
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\EmailEditor\Engine\SettingsController;
+
+abstract class BlockRenderer {
```

Comment:
I would rather avoid adding abstract classes if possible. It makes it harder for extenders to construct renderer because they need to somehow get the `$settingsController`. Please see my other comment in RendrererRegistry